README-dev.md

```diff
@@ -26,19 +26,19 @@ Quick start instructions:
     git clone git@github.com:MinaProtocol/mina.git
     ```
 
-If you have already done that, remember that the MinaProtocol and o1-labs repositories do not accept the password authentication used by the https URLs. You must set GitHub repos to pull and push over ssh: 
+    If you have already done that, remember that the MinaProtocol and o1-labs repositories do not accept the password authentication used by the https URLs. You must set GitHub repos to pull and push over ssh:
 
     ```sh
     git config --global url.ssh://git@github.com/.insteadOf https://github.com/
     ```
 
-3.  Pull in the submodules: 
+3.  Pull in the submodules:
 
     ```sh
     git submodule update --init --recursive
     ```
 
-    If this command fails with `git@github.com: Permission denied (publickey).` then you need to set up SSH keys on your machine. Follow the [Generating a new SSH key and adding it to the ssh-agent](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) instructions.
+If this command fails with `git@github.com:Permission denied (publickey).` then you need to set up SSH keys on your machine. Follow the [Generating a new SSH key and adding it to the ssh-agent](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) instructions.
```

Comment:
I don't think you want to remove the indentation here.

Reply:
Good catch!